CONTRIBUTING.md

```diff
@@ -1,4 +1,9 @@
-# Request for contributions
+# Contributing to OSEM
+Our open source project is open to contributions. Open your [issue](https://github.com/openSUSE/osem/issues/new) and send a PR!
```

Comment:
It is not about first open an issue and then sending a PR. I want to avoid that people (especially GSoC students in preparation for GSoC) open random PR. Instead I want an issue, discussing in the issue if we want to implement it and finally a PR after we agreed on sth in the issue.